src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,141 @@
+import { FaInstagram, FaWhatsapp, FaEnvelope } from 'react-icons/fa';
+import styles from './Footer.module.scss';
+import type { ReactNode } from 'react';
+
+export interface CommentData {
+  comment: string;
+  name: string;
+  phone: string;
+}
+
+export interface FooterProps {
+  brandName?: string[];
+  workingHours?: string[];
+  contacts?: {
+    type: 'instagram' | 'whatsapp' | 'email';
+    href: string;
+    label?: string;
+  }[];
+  highlightText?: string;
+  commentTitle?: string;
+  className?: string;
+  onSendComment?: (data: CommentData) => void;
+}
+
+const iconByType: Record<'instagram' | 'whatsapp' | 'email', ReactNode> = {
+  instagram: <FaInstagram />,
+  whatsapp: <FaWhatsapp />,
+  email: <FaEnvelope />,
+};
+
+export const Footer = ({
+  brandName = ['Se', 'Doce', 'Fosse'],
+  workingHours = ['Seg-Sex | 8h-18h', 'sáb | 8h-16h', 'Dom | Fechado'],
+  contacts = [
+    { type: 'instagram', href: '#', label: 'Instagram' },
+    { type: 'whatsapp', href: '#', label: 'Whatsapp' },
+    { type: 'email', href: '#', label: 'E-mail' },
+  ],
+  highlightText = 'Queremos saber sua opinião!   Avalie nossa loja!',
+  commentTitle = 'Deixe um Comentário',
+  className = '',
+  onSendComment,
+}: FooterProps) => {
+  const handleSubmit: React.FormEventHandler<HTMLFormElement> = (e) => {
+    e.preventDefault();
+    if (!onSendComment) return;
+
+    const form = e.currentTarget;
+    const formData = new FormData(form);
+
+    onSendComment({
+      comment: String(formData.get('comment') || ''),
+      name: String(formData.get('name') || ''),
+      phone: String(formData.get('phone') || ''),
+    });
+    form.reset();
+  };
+
+  return (
+    <footer
+      className={`${styles.footer || ''} ${className}`}
+      data-testid="site-footer"
+      aria-labelledby="footer-brand"
+    >
+      <div className={styles.brand || ''}>
+        <h2 id="footer-brand">
```

Comment:
ao inves de usar o h2, baixa a imagem que está no figma, até porque a fonte que eles usam varia de palavra pra palavra é dificil reproduzir assim na mão. Só tem que abrir o [componente do footer no figma](https://www.figma.com/design/0X6cHoK6Z9emFbXRTFpx8G/Mockups?node-id=649-2426&t=GjDzcZHTfFj8Ter2-1) , clicar na imagem e no canto direito tem a opção exportar. Daí salva com algum nome tipo "logo-footer" e adiciona numa pasta dentro de src. Pode criar uma pasta para as images, faz esse caminho: `src/assets/images`

Reply:
dai importa e usa com o componente <img>
`import logoFooter from '../../assets/images/logo-footer.png';`
```suggestion
      <img
            src={logoFooter}
            alt="logo se fosse doce"
            className={styles.logo || ''}
          />
```